tests/page-objects/default/enketo/pregnancy.wdio.page.js

```diff
@@ -90,130 +60,66 @@ const setFutureVisitDate = async (value = moment().add(1, 'day').format('YYYY-MM
   await date.setValue(value);
 };
 
-const selectYesNoOption = async (selector, value = 'yes') => {
-  const element = await $(`${FORM} ${selector}[value="${value}"]`);
-  await element.waitForDisplayed();
-  await element.click();
-  return value === 'yes';
-};
-
-// The selector for the 'riskFactors' is different depending on the option selected in the 'first pregnancy' question
-const selectAllRiskFactors = async (value = FIRST_PREGNANCY_VALUE.no) => {
-  const cbRisks = await riskFactors(value);
-  for (const risk of cbRisks) {
-    if (await risk.getAttribute('value') !== 'none') {
-      await risk.click();
-    }
-  }
-  return cbRisks.length - 1; // Subtract 1 due to the 'none' option
-};
-
-const getSummaryDetails = async () => {
-  return {
-    patientNameSumm: await patientNameSummary().getText(),
-    weeksPregnantSumm: await weeksPregnantSummary().getText(),
-    eddSumm: await eddSummary().getText(),
-    riskFactorsSumm: await riskFactorsSummary().length,
-    dangerSignsSumm: await dangerSignsSummary().length,
-  };
-};
-
-const submitPregnancy = async ({
-  gestationalAge: gestationalAgeValue = GESTATION_AGE.edd,
-  deliveryDate: deliveryDateValue = moment().add(1, 'month').format('YYYY-MM-DD'),
-  ancVisitPast: ancVisitPastValue = 0,
-  knowFutureVisits: knowFutureVisitsValue = 'yes',
-  futureVisitDate: futureVisitDateValue = moment().add(1, 'day').format('YYYY-MM-DD'),
-  firstPregnancy: firstPregnancyValue = 'no',
-  miscarriage: miscarriageValue = 'yes',
-  firstPregnancySelected: firstPregnancySelectedValue = FIRST_PREGNANCY_VALUE.no,
-  aditionalFactors: aditionalFactorsValue = 'no',
-  vaginalBleeding: vaginalBleedingValue = 'yes',
-  fits: fitsValue = 'yes',
-  abdominalPain: abdominalPainValue = 'yes',
-  headache: headacheValue = 'yes',
-  veryPale: veryPaleValue = 'yes',
-  fever: feverValue = 'yes',
-  reduceFetalMov: reduceFetalMovValue = 'yes',
-  breakingWater: breakingWaterValue = 'yes',
-  easilyTired: easilyTiredValue = 'yes',
-  swellingHands: swellingHandsValue = 'yes',
-  breathlessness: breathlessnessValue = 'yes',
-  llin: llinValue = 'yes',
-  ironFolate: ironFolateValue = 'yes',
-  dewormingMedication: dewormingMedicationValue = 'yes',
-  hivTested: hivTestedValue = 'yes'
-} = {}) => {
+const submitDefaultPregnancy = async () => {
   await commonPage.openFastActionReport('pregnancy');
-  await selectGestationAge(gestationalAgeValue);
+  //await selectGestationAge(GESTATION_AGE.edd);
```

Comment:
Why do we not have to select gestational age anymore?

Reply:
We have to select it, it is just that I am using the method that I created for the radioButtons

``` .js
await commonEnketoPage.selectRadioButton('How would you like to report the pregnancy?',  'Expected date of delivery (EDD)');
```

I need to delete that comment line, thanks!